sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/FromQuery.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.DigitalTwins.Core.Queries.QueryBuilders;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// Query object that already contains a select clause.
+    /// </summary>
+    public class FromQuery : QueryBase
+    {
+        private WhereQuery _innerQuery;
```

Comment:
I think we should call this `_delegationClause` as in, the API can be continued by this Query moving forward. does that make sense?

Reply:
That makes sense. Should I rename _innerQuery in all the other files where it's used as well?

Reply:
yes, and the corresponding field in the other Query classes.